health-education-research.csl

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" page-range-format="minimal" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Health Education Research</title>
+    <id>http://www.zotero.org/styles/health-education-research</id>
+    <link href="http://www.zotero.org/styles/health-education-research" rel="self"/>
+    <link href="http://www.zotero.org/styles/elsevier-vancouver" rel="template"/>
+    <link href="" rel="documentation"/>
```

Comment:
@CRMTH, please add the "documentation" link here. Seem like it should be http://www.oxfordjournals.org/our_journals/healed/for_authors/index.html.